release/Earthfile

```diff
@@ -133,6 +107,112 @@ signed-release:
+    ARG COMMIT_SHA_SHORT="$EARTHLY_GIT_SHORT_HASH"
+
+    # The release tag (vX.X.X) that we're running for
+    ARG --required RELEASE_TAG
+
+
+    # ghcr.io/earthbuild/earthbuild:buildkitd-staging-8f307bf7 -> docker.io/earthbuild/buildkitd:${RELEASE_TAG}
+    BUILD +copy-img-to-dockerhub --IMG_NAME="buildkitd" --SRC_TAG="buildkitd-staging-${COMMIT_SHA_SHORT}" --RELEASE_TAG="${RELEASE_TAG}"
+    # ghcr.io/earthbuild/earthbuild:8f307bf7 -> docker.io/earthbuild/earthbuild:${RELEASE_TAG}
+    BUILD +copy-img-to-dockerhub --IMG_NAME="earthbuild" --SRC_TAG="${COMMIT_SHA_SHORT}" --RELEASE_TAG="${RELEASE_TAG}"
+    
+    # ghcr.io/earthbuild/earthbuild:buildkitd-staging-8f307bf7-ticktock -> docker.io/earthbuild/buildkitd-ticktock:${RELEASE_TAG}
+    BUILD +copy-img-to-dockerhub --IMG_NAME="buildkitd-ticktock" --SRC_TAG="buildkitd-staging-${COMMIT_SHA_SHORT}-ticktock" --RELEASE_TAG="${RELEASE_TAG}"
+    # ghcr.io/earthbuild/earthbuild:8f307bf7-ticktock -> docker.io/earthbuild/earthbuild-ticktock:${RELEASE_TAG}
+    BUILD +copy-img-to-dockerhub --IMG_NAME="earthbuild-ticktock" --SRC_TAG="${COMMIT_SHA_SHORT}-ticktock" --RELEASE_TAG="${RELEASE_TAG}"
+
+copy-img-to-dockerhub:
+    ARG --required IMG_NAME
+    ARG --required RELEASE_TAG
+    ARG --required SRC_TAG
+    
+    LET SRC_DOCKER_REGISTRY="ghcr.io"
+    LET DEST_DOCKER_REGISTRY="docker.io"
+    
+    LET SRC_IMG="${SRC_DOCKER_REGISTRY}/earthbuild/earthbuild:${SRC_TAG}"
+    LET DEST_IMG="${DEST_DOCKER_REGISTRY}/earthbuild/${IMG_NAME}:${RELEASE_TAG}"
+
+    BUILD +copy-img --SRC_DOCKER_REGISTRY="${SRC_DOCKER_REGISTRY}" \
+        --DEST_DOCKER_REGISTRY="${DEST_DOCKER_REGISTRY}" \
+        --SRC_REF="${SRC_IMG}" --DEST_REF="${DEST_IMG}"
+
+copy-img:
+    FROM +skopeo
+    ARG --required SRC_REF
+    ARG --required DEST_REF
+    ARG DOCKER_REGISTRY="182221462027.dkr.ecr.us-west-2.amazonaws.com"
+
+    ARG SRC_DOCKER_REGISTRY="${DOCKER_REGISTRY}"
+    ARG DEST_DOCKER_REGISTRY="${DOCKER_REGISTRY}"
+
+    RUN --push --no-cache echo "Copying image from '${SRC_REF}' to '${DEST_REF}'"
+
+    ARG --required DOCKER_REGISTRY_USER
+    RUN --secret DOCKER_REGISTRY_TOKEN=DOCKER_REGISTRY_TOKEN skopeo login --username "${DOCKER_REGISTRY_USER}" --password "${DOCKER_REGISTRY_TOKEN}" "${SRC_DOCKER_REGISTRY}"
+
+    IF [ "$SRC_DOCKER_REGISTRY" != "$DEST_DOCKER_REGISTRY" ]
+        ARG --required DEST_DOCKER_REGISTRY_USER
+        RUN --secret DEST_DOCKER_REGISTRY_TOKEN=DOCKER_REGISTRY_TOKEN skopeo login --username "${DEST_DOCKER_REGISTRY_USER}" --password "${DEST_DOCKER_REGISTRY_TOKEN}" "${DEST_DOCKER_REGISTRY}"
+    END
+
+    RUN --push --no-cache skopeo copy --all --insecure-policy docker://${SRC_REF} docker://${DEST_REF}
+
+    RUN --push --no-cache echo "Done copying image from '${SRC_REF}' to '${DEST_REF}'"
+
+skopeo:
+    FROM alpine:3.20
+
+    # https://github.com/containers/skopeo/blob/main/install.md#alpine
+    RUN apk add skopeo
+
 release-homebrew:
```

Comment:
@janishorsts is any of this logic useful or should we just delete the `release-homebrew` target?

Reply:
Sorry for not getting back to you sooner!

Yes, delete it.

We can decouple this repo from the homebrew (or other distribution repos) based on a release signal (renovate etc). Keeps this space nice and clean.